dashboard/apps/moj_admin/templates/moj_admin/base.html

```diff
@@ -0,0 +1,114 @@
+{% extends 'moj_admin/moj_base.html' %}
+
+{% load staticfiles %}
+
+{% block top_of_page %}
+    {% load i18n admin_static %}
+{% endblock %}
+
+{% block page_title %}
+
+    MoJ Product Dashboard
+
+{% endblock %}
+
+{% block stylesheets %}
+
+    <link rel="stylesheet" type="text/css" href="{% static "moj_admin/css/dwp_admin.css" %}"/>
+
+    {% block extrastyle %}
+
+    {% endblock %}
+{% endblock %}
+
+{% block head %}
+    {% block blockbots %}
+        <meta name="robots" content="NONE,NOARCHIVE"/>
+    {% endblock %}
+    <script type="text/javascript" src="https://code.jquery.com/jquery-2.2.4.min.js"></script>
```

Comment:
It's the same as in site-packages/moj_template/templates/moj_template/base.html. I had to modify this template and store it as a new file, as it shared some tags with the base django admin template ('content' for example, possibly others). Ideally (I think?) we'd propose the jquery issue and other changes to the moj-template package but happy to deal with as you both think best. Let me know.